dicom_standard/parse_lib.py

```diff
@@ -29,7 +31,7 @@
 ID_PATTERN = re.compile(r'\b(' + '|'.join(NONSTANDARD_SECTION_IDS) + r').+\b')
 SHORT_DICOM_URL_PREFIX = "http://dicom.nema.org/medical/dicom/current/output/chtml/part03/"
 
-allowed_attributes = ["href", "src", "type", "data", "colspan", "rowspan"]
+JsonDataType = Union[List[MetadataTableType], MetadataTableType]
```

Comment:
I wasn't really sure what to name it since the JSON file can contain either a list of dicts or a dict. I feel like this is the most accurate name for the type unless we use an extremely long name to convey that the type is either a list of `MetadataTableType` or a singular `MetadataTableType`. 

I also didn't want to adjust the name `MetadataTableType` because it already accurately represents what the table contains. `TableDictType` already exists in `table_utils` and is defined as `Dict[str, str]`.